### PR TITLE
Transitions

### PR DESCRIPTION
**Pull request by andriignedash**

### CODE REVIEW

**Comment by andriignedash** [comment]
[Transition Check. Fixes](https://trello.com/c/BRzhTD5u/18-transition-check-fixes)

File: src/css/common/buttons.css [hunk 1]
```diff
@@ -23,6 +23,7 @@
 .btn-primary {
```
Comment (VOSolyanik) [inline src/css/common/buttons.css:22]: remove transition from `.btn` then

File: src/css/components/social-links.css [hunk 2]
```diff
@@ -17,7 +17,7 @@
   align-items: center;
   width: 42px;
   height: 42px;
-  transition: border-color var(--transition-duration) var(--transition-function);
+  transition: border var(--transition-duration) var(--transition-function);
```
Comment (VOSolyanik) [inline src/css/components/social-links.css]: lets leave `border-color`

File: src/css/sections/footer.css [hunk 3]
```diff
@@ -57,16 +57,16 @@
 .footer .nav-logo {
   margin-top: -8px;
   color: var(--white-color);
+  transition: color var(--transition-duration) var(--transition-function);
```
Comment (VOSolyanik) [inline src/css/sections/footer.css]: not needed as added on `.nav-logo `

File: src/css/sections/footer.css [hunk 4]
```diff
@@ -57,16 +57,16 @@
...
 .footer .nav-logo:hover,
 .footer .nav-logo:focus {
   color: var(--green-color);
 }
 
-.footer .nav-logo:hover .accent,
-.footer .nav-logo:focus .accent {
-  color: var(--white-color);
+.accent {
+  transition: color var(--transition-duration) var(--transition-function);
```
Comment (VOSolyanik) [inline src/css/sections/footer.css]: revert this change, and we also have tharnsition on `.nav-logo .accent`

File: src/css/sections/footer.css [hunk 5]
```diff
@@ -94,6 +94,7 @@
   font-weight: 600;
   color: var(--white-color);
   text-decoration: none;
+  transition: text-decoration var(--transition-duration) var(--transition-function);
```
Comment (VOSolyanik) [inline src/css/sections/footer.css]: I think transition not working with `text-decoration`

File: src/css/sections/footer.css [hunk 6]
```diff
@@ -167,6 +168,7 @@
   line-height: 1.3;
   color: rgba(var(--white-color-rgb), 0.6);
   text-decoration: none;
+  transition: text-decoration var(--transition-duration) var(--transition-function);
```
Comment (VOSolyanik) [inline src/css/sections/footer.css]: I think transition not working with `text-decoration`

**Review by VOSolyanik — CHANGES_REQUESTED** [review]

File: src/css/sections/footer.css [hunk 7]
```diff
@@ -69,11 +69,6 @@
   color: var(--white-color);
 }
 
-.footer .nav-logo:hover .accent,
```
Comment (VOSolyanik) [inline src/css/sections/footer.css:72]: revert this back

**Review by VOSolyanik — CHANGES_REQUESTED** [review]

**Review by VOSolyanik — APPROVED** [review]
